clang/lib/AST/Type.cpp

```diff
@@ -3413,8 +3413,8 @@ StringRef BuiltinType::getName(const PrintingPolicy &Policy) const {
     return "reserve_id_t";
   case IncompleteMatrixIdx:
     return "<incomplete matrix index type>";
-  case OMPArraySection:
-    return "<OpenMP array section type>";
+  case ArraySection:
+    return "<array section type>";
```

Comment:
Generalized again, but this is one I couldn't really do anything about, all I have is the 'type', not the associated expression.

Reply:
Ok